Unity/Assets/Scripts/MyTouchObserver.cs

```diff
@@ -98,6 +101,7 @@ public class MyTouchObserver : MonoBehaviour, ISingleTouchObserver {
         }
 
         void ReactTouchIn() {
+            Debug.Log("touch started");
             this.Transform.localScale = Vector3.one * 1.2f;
```

Comment:
You may want to avoid scaling in `z`.  Often we'll use the `Layer` script (in `SagoUtils.Depth`) to manage the `z` depth of objects, and scaling in `z` can modify the absolute position (i.e. if there is a non-zero `localPosition.z`, which there usually is).

Reply:
Ok! Just for this project I'm just using rotated planes, so the object's axis also aren't correct and they have to be z-scaled.